### ISSUE TITLE
Pagination in reactivesearch using ResultList component and 508 compliance

### ISSUE THREAD

**Issue by allendol** [appbaseio/reactivesearch#448]
We are using pagination in search result. We use ResultList component. Additional to click on page 2, 3 and to support 508 compliance, we need to use tab in order to go to different pages .
How can we do it?

**Comment by divyanshu013** [comment]
This will need some a11y improvements across the components. Related https://github.com/appbaseio/reactivesearch/issues/179

**Comment by allendol** [comment]
Does this mean using tab between search result pages currently is not possible?

**Comment by divyanshu013** [comment]
Yes, currently it won't work with tab key.

**Comment by metagrover** [comment]
Open to accepting PRs on this. If you or anyone wants to take this up, we can start with improving the accessibility of the  [Pagination buttons here](https://github.com/appbaseio/reactivesearch/blob/dev/packages/web/src/components/result/addons/Pagination.js).

**Comment by allendol** [comment]
I am new to react. I have used 
import './Pagination.js'; 
to include the file in app.js. I don't see any additional buttons and existing pagination stays as is.
Do I need to disable or enable some parameter?

**Comment by allendol** [comment]
Any feedback? Pagination should be 508 compliance with any possible work around. Please give the solution as detail and as clear as possible.

**Comment by divyanshu013** [comment]
@allendol the `Button` component would have to be focus-able so they can be cycled via tab key (`tabIndex="0"`). Also when a button is in focus it should show some outline (box-shadow) in a darker shade of the primary color preferably. Also it would be best to check if it breaks the UI in other places where the `Button` component is used. For  If it sounds a bit complex, I can take a look at this issue but it might be a while before I get to this.

For reference - https://ant.design/components/pagination/ (we could use a box-shadow too to make the focus clearer) :)
